eventuate-core/src/main/scala/com/rbmhtechnology/eventuate/log/EventLog.scala

```diff
@@ -605,6 +605,19 @@ abstract class EventLog[A <: EventLogState](id: String) extends Actor with Event
 }
 
 object EventLog {
+
+  /**
+   * Published on the event-stream, when the [[CircuitBreaker]] closes as a write finally
+   * was successful (after n retries).
+   */
+  case class EventLogAvailable(logId: String)
```

Comment:
Good idea. Implemented the changes in another [commit](https://github.com/RBMHTechnology/eventuate/pull/293/commits/2fb64d62f6eff14d61cd0ca199f5469fa2f1a0c0). Maybe you could have another look. Thanks.